server/test/integration/BraveAlerterConfiguratorTest/getAlertSessionByPhoneNumberTest.js

```diff
@@ -18,6 +18,7 @@ describe('BraveAlerterConfigurator.js integration tests: getAlertSessionByPhoneN
     this.installationIncidentCategories = ['Cat1', 'Cat2', 'Cat3']
 
     await db.clearSessions()
+    await db.clearNotifications()
```

Comment:
For all the tests where this was added in the `beforeEach`, could you please also add it in the `afterEach`?